components/automate-ui/src/app/components/calendar/calendar.component.ts

```diff
@@ -25,7 +25,7 @@ export class CalendarComponent {
 
```

Comment:
Storing a moment for each day on the calendar. When clicked the year, month, and day are returned as '2019-10-23'.